zip-0000.rst

```diff
@@ -557,7 +622,8 @@ status, upon request by any person, if they have not made progress in
 one year. Such a ZIP may be changed to Draft status if the Owner
 provides revisions that meaningfully address public criticism of the
 proposal, or to Proposed status if it meets the criteria required as
-described in the previous paragraphs.
+described in the previous paragraphs. Rejections are applied by consensus
+among the current ZIP Editors.
 
 A Consensus or Standards ZIP becomes Final when its associated network
 upgrade or other protocol change is activated on Zcash's mainnet.
```

Comment:
s/may/MAY/ two lines below, and 12 lines below.

Reply:
Also s/must/MUST/ in the next two paragraphs.

Reply:
Both "Updated by" and "Updated-by" spellings are used for that header. Other header fields use the hyphenated form.